src/samples.jl

```diff
@@ -117,5 +117,98 @@ function _sample_unit_nball_muller!(D::Vector{Vector{N}}, n::Int, p::Int;
     return D
 end
 
+## Rejection Sampling
+
+"""
+    _canonical_length(X::LazySet)
+
+Get the canonical length of the box-approximation of the convex set `X`.
```

Comment:
I had to look at the code to understand the meaning of "canonical length"; maybe say that the function `Returns the support function of the given set along the positive and negative canonical directions.`